features/wasm-non-trapping-float-to-int.yml

```diff
@@ -0,0 +1,7 @@
+name: Non-trapping float-to-int conversion (WebAssembly)
+description: Floating-point to integer conversion operators which saturate instead of trapping (on positive or negative overflow, they return the maximum or minimum integer value, respectively, and do not trap.)
```

Comment:
```suggestion
description: Saturating floating-point to integer conversion operators return the maximum or minimum integer value on overflow instead of trapping.
```